python/rapids/worker.py

```diff
@@ -67,10 +71,54 @@ def initialize_gpu_mem():
         pass
 
 
-if __name__ == '__main__':
-    # GPU context setup
-    initialize_gpu_mem()
+def handle_worker_exception(
```

Comment:
any chance to simply import the function from Spark https://github.com/apache/spark/blob/18aebd8eb86b554e7aab38baca1e5de24df19a57/python/pyspark/util.py#L473

Reply:
Unfortunately, this function was introduced pretty recently, 4.x IIRC. We cannot import it in old versions.